news/2020-08-03-mappers-report-july.md

```diff
@@ -0,0 +1,178 @@
+
+As for results, results for both May and June's MBC have been announced, including the announcement of the first winner of the MBC grand prize!
+
+May's contest concluded with 2 winners! Congratulations to both [hypercyte](https://osu.ppy.sh/users/9155377) who won both the community vote and "screening crew's favourite" categories, and to [Acylica](https://osu.ppy.sh/users/1943309) who was the community vote's runner-up! Further congratulations to [hypercyte](https://osu.ppy.sh/users/9155377) on being the first mapper to have won the MBC grand prize and earning the **Elite Mapper** title through the new Monthly Beatmapping Contests with an overall total of 7 points!
+
+As for June, we congratulate [Kalibe](https://osu.ppy.sh/users/3376777), [xbopost](https://osu.ppy.sh/users/6842421) and [Sylas](https://osu.ppy.sh/users/3906405) on winning MBC for this month! There was a 3-way tie this time so there will be no runner-up for this month. The scores between their entries were so thin with an amazing display of work from all three of them. [Mao](https://osu.ppy.sh/users/2204515) and [K4L1](https://osu.ppy.sh/users/11334594) also received honorable mentions for their creative entries which unfortunately did not score as high.
+
+View the latest information regarding MBC on the [wiki article](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest).
+
+### osu! Beatmapping World Championship 2020
+
+The osu! Beatmapping World Championship has progressed since last month and the round-of-16 matches have just concluded! Some results surprised everyone who were present during the results stream on Thursday. You can view the current results for the elimination rounds [here](https://obwc.net/results/elimination).
+
+#### Round of 16 results
+
+As mentioned, many of these results blew everyone away. Special shout-out to **Team Argentina** who showed everyone they're not messing around, even against the 6.9% of predictions in the RO16 [pick'ems](https://pickem.hwc.hr/tournaments/30/pick)!
+
+![](/wiki/shared/flag/FR.gif) France **5 - 0** Italy ![](/wiki/shared/flag/IT.gif)
+
+![](/wiki/shared/flag/CN.gif) China **2 - 3** Argentina ![](/wiki/shared/flag/AR.gif)
+
+![](/wiki/shared/flag/VN.gif) Vietnam **3 - 2** Canada ![](/wiki/shared/flag/CA.gif)
+
+![](/wiki/shared/flag/SE.gif) Sweden **2 - 3** Brazil ![](/wiki/shared/flag/BR.gif)
+
+![](/wiki/shared/flag/GB.gif) United Kingdom **4 - 1** United States ![](/wiki/shared/flag/US.gif)
+
+![](/wiki/shared/flag/AU.gif) Australia **3 - 2** Norway ![](/wiki/shared/flag/NO.gif)
+
+![](/wiki/shared/flag/KR.gif) South Korea **3 - 2** Denmark ![](/wiki/shared/flag/DK.gif)
+
+![](/wiki/shared/flag/PL.gif) Poland **2 - 3** Russian Federation ![](/wiki/shared/flag/RU.gif)    
+
+#### Quarterfinals Match-ups
+
+![](/wiki/shared/flag/FR.gif) France **vs** Argentina ![](/wiki/shared/flag/AR.gif)
```

Comment:
here as well, a table would also be very fitting for convenience

| Team 1 |   | Team 2 |
| --: | :-: | :-- |
| ![](/wiki/shared/flag/FR.gif) **France** | vs. | **Argentina** ![](/wiki/shared/flag/AR.gif) |
| ![](/wiki/shared/flag/VN.gif) **Vietnam** | vs. | **Brazil** ![](/wiki/shared/flag/BR.gif) |
| ![](/wiki/shared/flag/GB.gif) **United Kingdom** | vs. | **Australia** ![](/wiki/shared/flag/AU.gif) |
| ![](/wiki/shared/flag/KR.gif) **South Korea** | vs. | **Russian Federation** ![](/wiki/shared/flag/RU.gif) |